examples/light-switch-app/light-switch-common/light-switch-app.matter

```diff
@@ -640,6 +640,13 @@ client cluster Groups = 4 {
   request struct ViewGroupRequest {
     INT16U groupId = 0;
   }
+
```

Comment:
zap-regen updated the light-switch-app, as well.

Reply:
That is a flake I think. Please re-run zap

Reply:
Looks like @woody-apple did that! All set.